extensions/sql-migration/src/api/azure.ts

```diff
@@ -96,12 +96,12 @@ export async function getMigrationController(account: azdata.Account, subscripti
 export async function getMigrationControllers(account: azdata.Account, subscription: Subscription, resourceGroupName: string, regionName: string): Promise<MigrationController[]> {
 	const api = await getAzureCoreAPI();
 	const host = `https://${regionName}.management.azure.com`;
-	const path = `/subscriptions/${subscription.id}/resourceGroups/${resourceGroupName}/providers/Microsoft.DataMigration/Controllers?api-version=2020-09-01-preview`;
+	const path = `/subscriptions/${subscription.id}/providers/Microsoft.DataMigration/Controllers?api-version=2020-09-01-preview`;
 	const response = await api.makeAzureRestRequest(account, subscription, path, azurecore.HttpRequestMethod.GET, undefined, true, host);
 	if (response.errors.length > 0) {
 		throw new Error(response.errors.toString());
 	}
-	return response.response.data;
+	return response.response.data.value;
```

Comment:
any concerns with null checking anything in the response payload prior to accessing?

Reply:
I doing most of the null checks at the callers. Sometime, receiving a null response is okay. So, it is up to the callers to define how they want to handle that situation.

Reply:
The return type for this doesn't indicate that undefined is a valid return value. If you want to have the callers handle it I would suggest updating the return type to `Promise<MigrationController[] | undefined>`

Reply:
I will make correct the types in a later iterations.